### PR TITLE
fix(0003-ank.md): change SKX to PKX

### PR DESCRIPTION
**Pull request by LesterKim**

### CODE REVIEW

**Review by jwerle — APPROVED** [review]

**Review by vipyne — APPROVED** [review]

File: text/0003-ank.md [hunk 1]
```diff
@@ -395,7 +395,7 @@ function generate0(Bp, Bs, S) {
   const D = blake2b(K.publicKey, 32)
   // pack Z = Zp | Zs
   const Z = concat([
-    pack0(SKX, D, Bp, K.publicKey),
+    pack0(PKX, D, Bp, K.publicKey),
```
[the remark this answers is not in the document]
Comment (LesterKim) [reply text/0003-ank.md]: @Prash74 This is just an example, not how we implement it.

[the remark this answers is not in the document]
Comment (LesterKim) [reply text/0003-ank.md]: For reference, see how `pack0` is defined in this RFC

[the remark this answers is not in the document]
Comment (Prash74) [reply text/0003-ank.md]: ahhh just saw the example, makes sense 👍

**Review by Prash74 — APPROVED** [review]
👍

**Pull request merged**